### PR TITLE
Added Python Solution for some questions

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
Proof of Acceptance
Screenshot from 2024-05-20 11-50-10

**Comment by freakin23** [comment]
Acceptance proof for cses 1745
Screenshot from 2024-05-20 12-59-12

File: solutions/gold/cses-1745.mdx [hunk 1]
```diff
@@ -123,4 +123,27 @@ public class MoneySums {
...
 </JavaSection>
 
+<PySection>
+```py
+n = int(input())
+money = list(map(int, input().split()))
+
+max_sum = n * 1000
+Dp = [[False] * (max_sum + 1) for _ in range(n + 1)]
+Dp[0][0] = True
```
Comment (SansPapyrus683) [inline solutions/gold/cses-1745.mdx:133]: rename to `dp`

File: solutions/silver/cses-1650.mdx [hunk 2]
```diff
@@ -103,4 +103,19 @@ public class RangeXorQueries {
 
 </JavaSection>
 
+<PySection>
```
Comment (SansPapyrus683) [inline solutions/silver/cses-1650.mdx:106]: please do different prs for different sols

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
yeah this is fine

File: solutions/silver/cses-1650.mdx (hunk 2 above)
Comment (freakin23) [reply solutions/silver/cses-1650.mdx:106]: I don't understand what you said here. Can you be specific?

Comment (freakin23) [reply solutions/silver/cses-1650.mdx:106]: Ok. I will do my best to come up with unique solutions

File: solutions/gold/cses-1745.mdx [hunk 3]
```diff
@@ -118,9 +114,34 @@ public class MoneySums {
...
 </JavaSection>
+<PySection>
+
+```py
+n = int(input())
+money = list(map(int, input().split()))
+
+max_sum = n * 1000
+dp = [[False] * (max_sum + 1) for _ in range(n + 1)]
+dp[0][0] = True
```
Comment (SansPapyrus683) [inline solutions/gold/cses-1745.mdx:131]: show proof of ac for this _updated_ solution

**Review by SansPapyrus683 — APPROVED** [review]
idk if i approved alr but whatever

File: solutions/gold/cses-1745.mdx (hunk 3 above)
Comment (freakin23) [reply solutions/gold/cses-1745.mdx:131]: Screenshot from 2024-05-21 22-38-34
